packages/server/src/comment/comment.resolver.ts

```diff
@@ -27,6 +29,11 @@ export class CommentResolver {
     return this.commentService.findByIds(comment.replies);
   }
 
+  @ResolveField('file', () => File)
```

Comment:
Will need to mark the result as nullable if the return type is `Promise<File | null>`

`@ResolveField('file', () => File, { nullable: true })`